ci/terraform/oidc/kms-policies.tf

```diff
@@ -207,4 +207,51 @@ resource "aws_iam_policy" "mfa_reset_storage_token_kms_signing_policy" {
   description = "IAM policy for managing KMS connection for a lambda which allows signing of MFA reset storage token payloads"
 
   policy = data.aws_iam_policy_document.mfa_reset_storage_token_kms_signing_policy_document.json
+}
+
+# Policy for MFA reset JAR lambda to access the key used to sign the storage token claims
```

Comment:
I'd suggest

`# Policy for mfa-reset-authorize lambda to access the key used to sign the storage token claim in the MFA reset JAR`

Reply:
Updated!